<?xml version="1.0"?>

<launch>
  <arg name="robot_name" default="r0"/>
  <arg name="stop_buttons_topics" default="stop_buttons.yaml"/>
  <arg name="base_constraints_topic" default="base_constraints"/>
  <arg name="scan" default="/r0/laser0/scan"/>
  
  <group ns="$(arg robot_name)">
    <node pkg="tuw_safety_constraints" type="tuw_safety_constraints_node" name="tuw_safety_constraints_node" output="screen" >
      <remap from="base_constraints" to="$(arg base_constraints_topic)"/>
      <remap from="scan" to="$(arg scan)"/>
      <rosparam command="load" file="$(find tuw_safety_constraints)/cfg/$(arg stop_buttons_topics)" />
    </node>
  </group>
</launch>
